packages/@react-aria/menu/docs/useMenu.mdx

```diff
@@ -158,9 +158,11 @@ function Menu(props) {
 function MenuItem({item, state, onAction}) {
   // Get props for the menu item element
   let ref = React.useRef();
+  let isDisabled = state.disabledKeys.has(item.key);
+
   let {menuItemProps} = useMenuItem({
     key: item.key,
-    isDisabled: item.isDisabled,
+    isDisabled: isDisabled,
```

Comment:
keep `isDisabled: isDisabled` or go for shorthand just `isDisabled` ?

Reply:
shorthand IMO